templates/contribute/index.md

```diff
@@ -41,7 +41,7 @@ To ensure this, we only commit changes to `master` that have passed CI tests, an
 
 While you're working on a new contribution to `mathlib`, you should do this on a different branch.
 It's okay to do this in your own fork of the `mathlib` repository,
-or you can introduce yourself on Zulip and ask for write access to non-`master` branches of the mathlib repository.
+or you can introduce yourself on Zulip and ask for write access to non-`master` branches of the mathlib repository, you can either make your own thread to introduce yourself, or ask for access in [this topic](https://leanprover.zulipchat.com/#narrow/stream/287929-mathlib4/topic/github.20permission).
```

Comment:
Perhaps `make your own thread to introduce yourself` should link to #newmembers as a place to create the topic?

Reply:
done